### PR TITLE
G auth

### PR DESCRIPTION
**Pull request by nohren**
crediential auth works, working out getting g auth database info client side

### CODE REVIEW

File: components/signUp.jsx [hunk 1]
```diff
@@ -1,55 +1,66 @@
...
+    // combine the current obj with new state property or set old one
+    setState({...state, [e.target.name]: e.target.value});
+  };
 
 
   const submitUser = () => {
-    //check that data is there
-    if (state.password1 !== state.password2) {
-      alert('Passwords don\'t match')
+    console.log(Number(state.phone));
```
Comment (Scott-Guinn) [inline components/signUp.jsx]: sneaky console log, can we omit?

File: components/signUp.jsx [hunk 2]
```diff
@@ -1,55 +1,66 @@
...
-  }
-
-
+    } else if (state.password1 !== state.password2) {
+      alert('Passwords don\'t match');
+      return;
+    } else if (state.phone.length !== 10 ||
+      typeof Number(state.phone) !== 'number') {
+      alert('Phone number is incorrect');
+    } else if (!state.email.includes('@') || !state.email.includes('.com')) {
```
Comment (Scott-Guinn) [inline components/signUp.jsx:42]: can we remove the .com check? Some email domains are .me or .edu or .org

File: components/signUp.jsx [hunk 3]
```diff
@@ -1,55 +1,66 @@
...
+      alert('please enter a valid email');
+    } else {
+      // if it looks good submit to api
+      axios({
+        method: 'POST',
+        url: 'api/newUser',
+        data: state,
+      })
+          .then((result) => {
+            console.log(result + 'this is the result.');
```
Comment (Scott-Guinn) [inline components/signUp.jsx]: console log

File: components/signUp.jsx [hunk 4]
```diff
@@ -1,55 +1,66 @@
...
+            alert('Account creation successful!');
+            setSignup(false);
+          })
+          .catch((err) => {
+            console.log(err);
```
Comment (Scott-Guinn) [inline components/signUp.jsx]: mind changing this to a console.error?

**Review by benkennedy98 — COMMENTED** [review]
Looks good

File: components/signUp.jsx [hunk 5]
```diff
@@ -124,26 +125,31 @@ const SignUp = () => {
 
 
         <Form.Group>
-          <Form.Label>Water Goal</Form.Label>
+          <Form.Label>Daily Water Goal</Form.Label>
```
Comment (Scott-Guinn) [inline components/signUp.jsx:128]: can we add "(in ounces)" so the user knows when they sign up

File: db/index.js [hunk 6]
```diff
@@ -150,14 +158,13 @@ const getUsernameID = (username) => {
 };
```
Comment (taylorsmart) [inline db/index.js:158]: Awesome.  Thank you!

File: schema.sql [hunk 7]
```diff
@@ -71,5 +71,8 @@ INSERT INTO quotes(failure_quote, success_quote) VALUES('Mistakes are inevitable
 INSERT INTO quotes(failure_quote, success_quote) VALUES('You will gain confidence with every small positive change you are making.', 'You''ve got this!');
 INSERT INTO quotes(failure_quote, success_quote) VALUES('You can accomplish anything you set your mind to!', 'Keep up the great work!');
 
+<<<<<<< HEAD
+=======
 INSERT INTO users(firstName,  lastName, password, email,  calorieGoal, waterGoal) VALUES('John', 'Doe', 'password', 'jdoe@yahoo.com', 5, 5);
```
Comment (taylorsmart) [inline schema.sql:75]: @nohren mind fixing this up?

File: schema.sql [hunk 8]
```diff
@@ -71,5 +71,8 @@ INSERT INTO quotes(failure_quote, success_quote) VALUES('Mistakes are inevitable
...
-INSERT INTO sessions(session,  user_id) VALUES('1', 1);
\ No newline at end of file
+INSERT INTO sessions(session,  user_id) VALUES('1', 1);
+>>>>>>> 0f3bc21ec5cf37202961a9dad1771d16e50a5701
```
Comment (Scott-Guinn) [inline schema.sql]: looks like merge conflict got committed

File: components/signUp.jsx (hunk 2 above)
Comment (taylorsmart) [reply components/signUp.jsx:42]: Leaving it in temporarily - there are really nice regex expressions for email validation.

**Review by taylorsmart — APPROVED** [review]

**Pull request merged**